cli/standalone/binary.rs

```diff
@@ -904,6 +906,46 @@ impl<'a> DenoCompileBinaryWriter<'a> {
   }
 }
 
+fn get_denort(deno_exe: PathBuf) -> Option<OsString> {
+  let mut denort = deno_exe;
+  denort.set_file_name("denort");
+  denort.exists().then(|| denort.into_os_string())
+}
+
+fn is_in_target(path: &Path) -> bool {
+  path
+    .components()
+    .any(|component| component == Component::Normal("target".as_ref()))
+}
+
+fn try_get_denort(path: &Path) -> Option<OsString> {
+  if is_in_target(path) {
+    get_denort(path.to_path_buf())
+  } else {
+    None
+  }
+}
+
+fn get_dev_binary_path() -> Option<OsString> {
+  env::var_os("DENORT_BIN").or_else(|| {
+    env::current_exe().ok().and_then(|exec_path| {
+      if is_in_target(&exec_path) {
+        try_get_denort(&exec_path)
+      } else if exec_path.is_symlink() {
+        fs::read_link(&exec_path).ok().and_then(|target_path| {
+          if is_in_target(&target_path) {
+            try_get_denort(&target_path)
+          } else {
+            None
+          }
+        })
+      } else {
+        None
```

Comment:
Do you think we should only do this for "debug" builds, or check that the path includes `target/debug`? Or is there value in providing this for release builds too?